styles.css

```diff
@@ -95,6 +114,11 @@ footer p {
   text-align: center;
 }
 
+.ratio-result p,
+.check-result p {
+  font-size: 1.15rem;
```

Comment:
Maybe for these two paragraphs:
```
.ratio-result p,
.check-result p
```
give them some class in HTML, so you can use some class selector instead of this one